.github/scripts/bugzilla_reference_check.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+from __future__ import unicode_literals
+
+import argparse
+import os
+import re
+import sys
+
+from bugzilla import Bugzilla
+import requests
+
+UH_OH = '''
+       _                 _     _
+ _   _| |__         ___ | |__ | |
+| | | | '_ \ _____ / _ \| '_ \| |
+| |_| | | | |_____| (_) | | | |_|
+ \__,_|_| |_|      \___/|_| |_(_)
+'''
+
+bugzilla_token = os.environ.get('BUGZILLA_TOKEN')
+if not bugzilla_token:
+    raise EnvironmentError('BUGZILLA_TOKEN not specified')
+
+github_token = os.environ.get('GITHUB_TOKEN_PSW')
+if not github_token:
+    raise EnvironmentError('GITHUB_TOKEN not specified')
+
+pr_number = os.environ.get('PR_NUMBER')
+if not github_token:
+    raise EnvironmentError('PR_NUMBER not specified')
```

Comment:
I modified this from the original check_pr_branch.py script. I'm setting the PR number through an environment variable instead of an argument in the script. I was having issues with passing in ${{ github.event.pull_request.number }} as an argument.